docs/guides/operator/installation.adoc

```diff
@@ -62,7 +64,21 @@ kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resourc
 kubectl apply -f https://raw.githubusercontent.com/keycloak/keycloak-k8s-resources/{version}/kubernetes/kubernetes.yml
 ----
 
-Currently the Operator watches only the namespace where the Operator is installed.
+The Operator will watch the namespace where it is installed. You may optionally select a namespace with the `-n` option. 
 </@profile.ifCommunity>
 
+=== Installing Multiple Operators
+
+It is currently not fully supported for the operator to watch multiple or all namespaces. In circumstances where you want to watch multiple namespaces, you can install multiple operators.
+
+If you do this please be aware:
+
+- all Operators share the CRDs (Custom Resource Definitions) as they are installed cluster wide.
+- CRD revisions from newer Operator versions won't introduce breaking changes except for the eventual removal of fields that have been well deprecated. Thus newer CRDs are generally backward compatible.
+- the CRDs installed last will be the ones in use.
+- older CRDs may not be forwards compatible with new fields used by newer operators. When using OLM it will check if your custom resources are compatible with the CRDs being installed, so the usage of new fields can prevent the simultaneous installation of older operator versions.
```

Comment:
```suggestion
- older CRDs may not be forwards compatible with new fields used by newer operators. When using OLM it will check if your custom resources are compatible with the CRDs being installed, so the usage of new fields can prevent the simultaneous installation of older operator versions. In that case you first need to install the older Operator version and then the newer one that introduces new fields to the CRDs.
```

Reply:
Yeah, that's a good point. I didn't realize the upgrades. This behaviour sucks more than I thought. I think the proper solution would be the cluster-wide Operator. As you suggested somewhere else, an alternative might also be to release the CRDs as an individual bundle but that feels more like a potentially confusing workaround to me.

Reply:
Right. We do also have the option of exposing our cluster wide support, but just document it as preview...

Reply:
@shawkins We'd need to first figure out our support matrix and also how to communicate to users which fields are supported with which Keycloak versions.

Reply:
The cluster wide support doesn't need to have those kinds of problems - if we simply remove the unsupported fields from the CRDs when support is dropped, then the users will be blocked from migrating to that version until they have dealt with the removal. However there needs to be an inbetween release that has both old and new support - which is what we're breaking wrt to hostname v1 and product users.